presidio-analyzer/presidio_analyzer/pattern_recognizer.py

```diff
@@ -22,7 +22,7 @@ class PatternRecognizer(LocalRecognizer):
 
     :param patterns: A list of patterns to detect
     :param deny_list: A list of words to detect,
-    in case our recognizer uses a predefined list of words (black list)
+    in case our recognizer uses a predefined list of words (deny list)
```

Comment:
This was here on purpose 😊 if somebody doesn't know thay deny list is the new black list...